src/commands/registerCommands.ts

```diff
@@ -36,7 +36,7 @@ export function registerCommands(refreshEventEmitter: vscode.EventEmitter<void>)
     registerCommand('azureResourceGroups.refresh', async (context: IActionContext, node?: AzExtTreeItem) => { await ext.appResourceTree.refresh(context, node); refreshEventEmitter.fire(); });
     registerCommand('azureResourceGroups.revealResource', revealResource);
     registerCommand('azureResourceGroups.selectSubscriptions', () => commands.executeCommand('azure-account.selectSubscriptions'));
-    registerCommand('azureResourceGroups.viewProperties', viewProperties);
+    registerCommandWithTreeNodeUnwrapping('azureResourceGroups.viewProperties', viewProperties);
```

Comment:
Does this necessarily need unwrapping, or can `BranchDataItem` just have `get` methods that reflect the necessary data?